packages/fx-core/tests/plugins/solution/solution.arm.test.ts

```diff
@@ -65,11 +66,13 @@ const simpleAuthPlugin = Container.get<Plugin>(ResourcePlugins.SimpleAuthPlugin)
 const spfxPlugin = Container.get<Plugin>(ResourcePlugins.SpfxPlugin) as Plugin & ArmResourcePlugin;
 const aadPlugin = Container.get<Plugin>(ResourcePlugins.AadPlugin) as Plugin & ArmResourcePlugin;
 
-const baseFolder = "./infra/azure";
+const baseFolder = "./templates/azure";
+const templatesFolder = "./templates";
 const parameterFolderName = "parameters";
-const templateFolderName = "templates";
+const templateFolderName = "modules";
+const configFolderName = "./.fx/configs";
+const parameterFileNameTemplate = "azure.parameters.@envName.json";
```

Comment:
I saw you use the env placeholder name from teamsfx-api in implementation, we may also use it here.

Reply:
done